Description: fix manpage extension for compatibility with Debian Perl Policy
Author: Damyan Ivanov <dmn@debian.org>
Origin: vendor
Forwarded: no
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -72,7 +72,7 @@
     $cc = can_cc unless $bork;
     my $par;
     $par = fetch_par('', '', !$cc) unless ($cc or $bork);
-    my $exe  = $Config::Config{_exe};
+    my $exe  = '';
     my $dynperl = $Config::Config{useshrplib} && ($Config::Config{useshrplib} ne 'false');
 
     if ($bork) {
@@ -115,11 +115,11 @@
 
     makemaker_args(
         MAN1PODS		=> {
-            'script/par.pl'	=> 'blib/man1/par.pl.1',
-            'script/pp'	        => 'blib/man1/pp.1',
-            'script/tkpp'       => 'blib/man1/tkpp.1',
+            'script/par.pl'	=> 'blib/man1/par.pl.1p',
+            'script/pp'	        => 'blib/man1/pp.1p',
+            'script/tkpp'       => 'blib/man1/tkpp.1p',
           ($par or $cc) ? (
-            'script/parl.pod'   => 'blib/man1/parl.1',
+            'script/parl.pod'   => 'blib/man1/parl.1p',
           ) : (),
         },
         EXE_FILES		=> [
